action/receipt_test.go

```diff
@@ -64,7 +73,12 @@ func TestConvert(t *testing.T) {
 
 func TestSerDer(t *testing.T) {
 	require := require.New(t)
-	receipt := &Receipt{1, 1, hash.ZeroHash256, 1, "", nil, nil, ""}
+	receipt := &Receipt{
+		Status:      1,
+		BlockHeight: 1,
+		ActionHash:  hash.ZeroHash256,
```

Comment:
same here

Reply:
this is an existing test, just write out the fields explicitly